.github/workflows/visual-ci-new.yaml

```diff
@@ -0,0 +1,36 @@
+name: web-new Visual Testing Pipeline
+
+on:
+    pull_request:
+        paths-ignore:
+            - "**.md"
+            - "**.png"
+            - "**.drawio"
+            - "**.xlsx"
+
+jobs:
+    setup-and-visual-test:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: Use Node.js 16
+              uses: actions/setup-node@v3
+              with:
+                  node-version: "16.x"
```

Comment:
Question: Does it make sense to use/rely-on v16 since it is out of [end of life](https://endoflife.date/nodejs) already?

Reply:
@SergioCasCeb has changed it to v18, resolving.